helpers/helper.py

```diff
@@ -16,13 +13,6 @@ def wrap(request):
                     logger.error(f"{field} is empty field")
                     return HttpResponseBadRequest(f"{field} not given")
 
-            # catch server exceptions
-            try:
```

Comment:
why not catch exception?

Reply:
Exception handler is already catching them, I don’t need to catch them elsewhere

Reply:
client get good response?

Reply:
No, code error is depend on catched exception